api/envoy/config/core/v3/protocol.proto

```diff
@@ -260,6 +260,15 @@ message HttpProtocolOptions {
   // a 431 response for HTTP/1.x and cause a stream reset for HTTP/2.
   google.protobuf.UInt32Value max_headers_count = 2 [(validate.rules).uint32 = {gte: 1}];
 
+  // The maximum combined size of headers.
```

Comment:
combined is a bit confusing IMO.  Maximum allowed size for request and response headers?

Reply:
What I'm trying to express is the sum of hte sizes of all the response headers (as opposed to a limit for the maximum size of a single header). I'll take another stab at wording.